### DISCUSSION TITLE
Reusing existing custom searches

### DISCUSSION THREAD

**Discussion by Jrstew2016** [microsoft-search/pnp-modern-search#3433] [Q&A] [answered]
What is the easiest way that we can re-use custom searches we've created that aren't on the same page or site? I know the custom template is easy. Just copy paste. But I seem to have had issues when copying and pasting them into the properties on a new site via the search result web part. Am I missing something? 
Or do we have to create our own package and deploy it in our environment? I'm not a developer so forgive me if my questions sound trivial! :)

**Comment by kasperbolarsen** [comment]
Hi
By copying you are referring to the Import/export option?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/5e54a0b8-98e3-4dc8-a0f8-1604db4cf32a)

**Reply by Jrstew2016** [reply]
I was actually just copying and pasting like you can for the custom templates. I'm assuming now though, I should be using the export, import. Correct? 
And if that's the case. Will it automatically populate the selected properties by checking the boxes as well as populate/fill out the layout slots?
And same goes for the Filter properties, too.
Thanks!

**Reply by kasperbolarsen** [reply, marked as answer]
It will indeed set the selected Properties, Layout slots and so on :-)
As far as customized display templates/Layouts I will recommend storing the HTML files in a central location in your tenant and just reference it using the "Use an external template url" :-)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/5bfc8a19-e7a9-4330-ba24-ce190ac27fe9)

**Reply by Jrstew2016** [reply]
If we use a central location to store the custom templates. Does every user have to have access to the directory in order for it to work on the site/page it's being displayed on? Or can we lock it down to just admins and it still render on a site it's being displayed on?

**Reply by kasperbolarsen** [reply]
Everybody will need read access to the display template library. The root site or similar would be a good choice. 

see https://microsoft-search.github.io/pnp-modern-search/scenarios/edit-templates-using-vscode-and-onedrive/#steps for tips and tricks

**Reply by Jrstew2016** [reply]
Great, thanks for all the info!
